lib/Jsrt/ChakraCore.h

```diff
@@ -679,5 +679,42 @@ CHAKRA_API
         _In_ JsPropertyIdRef propertyId,
         _Out_ bool *hasOwnProperty);
 
+/// <summary>
+///     Write JS string value into char string buffer without a null terminator
```

Comment:
I had addressed that at https://github.com/Microsoft/ChakraCore/pull/3408/files#diff-ffd84896ce29f73bbba0ed9612cf37b2R699 unless you feel that is unclear?